frontend/src/components/placementVisuals/AddPlacementContainer.tsx

```diff
@@ -130,8 +130,18 @@ const AddPlacementContainer = ({ onConfirm, destinationContainerList, setDestina
           else {
             containerAlreadyExists(container, destinationContainerList).then(exists => {
               if (!exists) {
-                addContainer(container)
-              }
+                if (container.container_barcode.includes(' ')) {
```

Comment:
ugh ... i would use a regular expression that matches the one in the backend ... Check barcodeRules and nameRules in ./src/constants.tsx and how they are used in the Sample edit form for example...